frontend/projects/upgrade/src/app/features/dashboard/home/components/metrics/metrics.component.ts

```diff
@@ -55,13 +57,18 @@ export class MonitoredMetricsComponent implements OnInit, OnChanges, OnDestroy {
   metricsDataSource = new BehaviorSubject<AbstractControl[]>([]);
 
   metricsDisplayedColumns = ['keys', 'operationType', 'queryName', 'removeMetric'];
-  queryIndex = 0;
+  queryIndex: number = 0;
+  editMode: Boolean = false;
```

Comment:
we want lowercase `boolean`. `Boolean` refers to the non-primitive `new Boolean()` object, which isn't quite right. https://stackoverflow.com/questions/15487220/typescript-primitive-types-any-difference-between-the-types-number-and-numbe

Reply:
Nice catch, will update this. Thanks